pages/add-clinic.vue

```diff
@@ -132,10 +143,26 @@ export default {
             .firestore()
             .collection("pending")
             .add(clinic)
-            .then(() => console.log("added to db", this.clinic))
-            .then(this.$router.push("/"));
+            .then(() => {
+              console.log("added to db", this.clinic)
+              this.snackBarData = {
```

Comment:
I like that you re-used the same notification instead of building two!

Reply:
Although, should we remove that console.log before merging?

Reply:
@renderf0x I haven't been commenting on `console.logs` since we don't have a proper logging system yet and they aren't helping on the user side.
BUT I was going to use them as points of find+replace targets when we get client side logging and they aren't hurting anything right now. 💡